utt/libutt/src/api/burn.cpp

```diff
@@ -6,7 +6,20 @@
 #include <utt/Params.h>
 #include <utt/Serialization.h>
 #include <utt/RegAuth.h>
+#include <ostream>
 
+std::ostream& operator<<(std::ostream& out, const libutt::api::operations::Burn& burn) {
+  out << *(burn.burn_) << std::endl;
+  out << burn.c_ << std::endl;
+  return out;
+}
+std::istream& operator>>(std::istream& in, libutt::api::operations::Burn& burn) {
+  in >> *(burn.burn_);
```

Comment:
why do you handle pointers differently ?  on paramters you perform ```params.params.reset(new libutt::Params());``` and here just override the values?

Reply:
In burn, the default constructor already initiates the pointer. With UTTParams we are unable to create globalParams object before initiating the underlying libraries. This is why we don't have a default constructor there